cms/core/blocks/embeddable.py

```diff
@@ -50,7 +50,7 @@ class DocumentBlock(blocks.StructBlock):
     title = blocks.CharBlock(required=False)
     description = blocks.RichTextBlock(features=settings.RICH_TEXT_BASIC, required=False)
 
-    class Meta:  # pylint: disable=missing-class-docstring,too-few-public-methods
+    class Meta:  # pylint: disable=missing-class-docstring
```

Comment:
This is unfortunate. I think we should (separately) try to put in place some global ignore rules for `Meta` classes.

Reply:
Perfect, let's do that.

Reply:
and.. that was a complete flop. it is either global, or per line, block or module :/

Reply:
I'm okay with it being global.

Reply:
c763430